### PR TITLE
Update limitation date; query parameters

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/group_delta.md [hunk 1]
```diff
@@ -18,6 +18,36 @@ To begin tracking changes, you make a request including the delta function on th
 ```http
 GET /groups/delta
 ```
+
+### Query parameters
+
+Tracking changes in messages incurs a round of one or more **delta** function calls. If you use any query parameter
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/group_delta.md]: You probably want to customize the first part of the first sentence here as "Tracking changes in _groups_ incurs..."

File: api-reference/beta/api/group_delta.md [hunk 2]
```diff
@@ -18,6 +18,36 @@ To begin tracking changes, you make a request including the delta function on th
...
+it in the initial **delta** request. Microsoft Graph automatically encodes any specified parameters 
+into the token portion of the `nextLink` or `deltaLink` URL provided in the response. 
+You only need to specify any desired query parameters once upfront. 
+In subsequent requests, simply copy and apply the `nextLink` or `deltaLink` URL from the previous response, as that URL already 
+includes the encoded, desired parameters.
+
+| Query parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+| $deltatoken | string | A [state token](../../../concepts/delta_query_overview.md) returned in the `deltaLink` URL of the previous **delta** function call for the same message collection, indicating the completion of that round of change tracking. Save and apply the entire `deltaLink` URL including this token in the first request of the next round of change tracking for that collection.|
+| $skiptoken | string | A [state token](../../../concepts/delta_query_overview.md) returned in the `nextLink` URL of the previous **delta** function call, indicating there are further changes to be tracked in the same message collection. |
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/group_delta.md]: Similar to the last comment - "... to be tracked in the same _group_ collection"?
Please apply the similar change to the other delta method topics you guys own.

File: api-reference/beta/api/group_delta.md (hunk 1 above)
Comment (PatAltimore) [reply api-reference/beta/api/group_delta.md]: Good catch.  Thanks!

**Comment by PatAltimore** [comment]
Publish to master: https://github.com/microsoftgraph/microsoft-graph-docs/pull/922

**Pull request closed without merging**